### PR TITLE
Fixed #8623 

### LINKED ISSUE #8623: Footer Copyright Year Displays Previous Year Instead of Current Year
**Describe the bug**
The copyright year in the footer of the sign-up, login, and forgot password pages is displaying the previous year instead of the current year.

**To Reproduce**
Steps to reproduce the behaviour:
1) Go to 'Shop'
2) Click the sign-up page, login page or Forgot Password Page
3) Scroll down to the footer of the above pages.
4) Observe that the copyright year is displaying the previous year.

**Expected behavior**
The copyright year in the footer of the sign-up, login, and forgot password pages should display the current year.

**Screenshots**
Please take Reference from [Here](https://paste.pics/350edb2675351879d8d37acff6b8c4bc)

### PR DESCRIPTION
**Pull request by JunaidJaved248**
## Issue Reference
#8623 
## Description
Fixed #8623  => Change @lang to trans for copyright footer message, & modified relevant lang files of shop, & Tested it working fine

## How To Test This?
Go to 'Shop'
Click the sign-up page, login page or Forgot Password Page
Scroll down to the footer of the above pages.
Observe that the copyright year is displaying now Current year.

### CODE REVIEW

**Comment by JunaidJaved248** [comment]
@amit-webkul Please review it

File: packages/Webkul/Shop/src/Resources/lang/ar/app.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
             'email-not-exist'      => 'لا يمكننا العثور على مستخدم بهذا البريد الإلكتروني',
             'email'                => 'البريد الإلكتروني',
             'forgot-password-text' => 'إذا نسيت كلمة المرور الخاصة بك، يمكنك استعادتها عن طريق إدخال عنوان البريد الإلكتروني الخاص بك.',
-            'footer'               => '© حقوق الطبع والنشر 2010 - 2022، Webkul Software (مسجلة في الهند). جميع الحقوق محفوظة.',
+            'footer'               => '© حقوق الطبع والنشر 2010 - :currentYear، Webkul Software (مسجلة في الهند). جميع الحقوق محفوظة.',
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/lang/ar/app.php:10]: current_year

File: packages/Webkul/Shop/src/Resources/views/customers/sign-up.blade.php [hunk 2]
```diff
@@ -229,7 +229,7 @@ class="primary-button block w-full max-w-[374px] py-[16px] px-[43px] mx-auto m-0
 		</div>
 
         <p class="mt-[30px] mb-[15px] text-center text-[#6E6E6E] text-xs">
-            @lang('shop::app.customers.signup-form.footer')
+            {{ trans('shop::app.customers.signup-form.footer',['currentYear'=> date('Y') ]) }}
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/sign-up.blade.php:232]: current_year

File: packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php [hunk 3]
```diff
@@ -106,13 +106,13 @@ class="primary-button block w-full max-w-[374px] m-0 ml-[0px] mx-auto px-[43px]
         </div>
 
         <p class="mt-[30px] mb-[15px] text-[#6E6E6E] text-xs text-center">
-            @lang('shop::app.customers.forgot-password.footer')
+            {{ trans('shop::app.customers.forgot-password.footer',['currentYear'=> date('Y') ]) }}
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php:109]: @lang everywhere inside blade files

Comment (JunaidJaved248) [reply packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php:109]: @devansh-webkul Sorry, I can't get it?

File: packages/Webkul/Shop/src/Resources/lang/ar/app.php (hunk 1 above)
Comment (JunaidJaved248) [reply packages/Webkul/Shop/src/Resources/lang/ar/app.php:10]: Okay @devansh-webkul , I will change it from currentYear to current_year.

File: packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php (hunk 3 above)
Comment (JunaidJaved248) [reply packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php:109]: Can you explain it bit, what you want to say? @devansh-webkul

Comment (devansh-webkul) [reply packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php]: @lang('shop::app.customers.forgot-password.footer', [...])

Comment (JunaidJaved248) [reply packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php]: Got it

**Comment by JunaidJaved248** [comment]
@devansh-webkul Check now

**Comment by JunaidJaved248** [comment]
> @devansh-webkul Check now

Have You checked it? Please review it.

**Review by Amitk-webkul — APPROVED** [review]

File: packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php [hunk 4]
```diff
@@ -106,13 +106,13 @@ class="primary-button block w-full max-w-[374px] m-0 ml-[0px] mx-auto px-[43px]
         </div>
 
         <p class="mt-[30px] mb-[15px] text-[#6E6E6E] text-xs text-center">
-            @lang('shop::app.customers.forgot-password.footer')
+            @lang('shop::app.customers.forgot-password.footer',['current_year'=> date('Y') ])
```
Comment (amit-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php:109]: Please add one space after the (, ) symbols in all blade files.

Comment (JunaidJaved248) [reply packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php]: @amit-webkul its done

**Review by amit-webkul — APPROVED** [review]

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**